### PR TITLE
spring starter now includes more resource providers

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

**Comment by zeitlinger** [comment]
@jeanbisutti please review

**Review by cartermp — APPROVED** [review]

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 1]
```diff
@@ -296,9 +296,14 @@ public class Application {
 
 #### Resource Providers
 
-The OpenTelemetry Starter includes
-[common Resource Providers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/resources/library)
-and the following Spring Boot specific Resource Providers:
+The OpenTelemetry Starter includes the same Resource Providers as the Java
```
Comment (theletterf) [inline content/en/docs/languages/java/automatic/spring-boot.md:299]:
```suggestion
The OpenTelemetry Starter includes the same resource providers as the Java
```

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 2]
```diff
@@ -296,9 +296,14 @@ public class Application {
...
+agent:
+
+- [common Resource Providers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/resources/library)
+- [Resource providers that are disabled by default](/docs/languages/java/automatic/configuration/#enable-resource-providers-that-are-disabled-by-default)
+
+In addition, the OpenTelemetry Starter includes the following Spring Boot
```
Comment (theletterf) [inline content/en/docs/languages/java/automatic/spring-boot.md:305]:
```suggestion
- [Common resource providers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/resources/library)
- [Resource providers that are disabled by default](/docs/languages/java/automatic/configuration/#enable-resource-providers-that-are-disabled-by-default)

In addition, the OpenTelemetry Starter includes the following Spring Boot
```

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 3]
```diff
@@ -296,9 +296,14 @@ public class Application {
...
+specific Resource Providers:
```
Comment (theletterf) [inline content/en/docs/languages/java/automatic/spring-boot.md:306]:
```suggestion
specific resource providers:
```

**Review by theletterf — APPROVED** [review]
There seems to be a capitalization issue in the whole doc...

**Comment by zeitlinger** [comment]
> There seems to be a capitalization issue in the whole doc...

do we have guidelines for which otel terms should be capitalized?

**Comment by cartermp** [comment]
Ah, crap, I hit the merge button before I realized you had asked @jeanbisutti to review.

Do you want to revert, or is this fine enough?

**Pull request merged**